wavesrv/pkg/cmdrunner/cmdrunner.go

```diff
@@ -1533,16 +1539,54 @@ func doOpenAIStreamCompletion(cmd *sstore.CmdType, opts *sstore.OpenAIOptsType,
 		}
 		sstore.MainBus.SendScreenUpdate(cmd.ScreenId, update)
 	}()
-	ch, err := openai.RunCompletionStream(ctx, opts, prompt)
+	var ch chan *packet.OpenAIPacketType
+	var err error
+	if opts.APIToken == "" {
+		clientData, err := sstore.EnsureClientData(ctx)
+		if err != nil {
+			writeErrorToPty(cmd, fmt.Sprintf("error getting client data: %v", err), outputPos)
+			return
+		}
+		if clientData.ClientOpts.NoTelemetry {
+			writeErrorToPty(cmd, fmt.Sprintf("In order to protect against abuse, you must have telemetry turned on in order to use Wave's free AI features.  If you do not want to turn telemetry on, you can still use Wave's AI features by adding your own OpenAI key in Settings.  Note that when you use your own key, requests are not proxied through Wave's servers and will be sent directly to the OpenAI API."), outputPos)
```

Comment:
Long message, should this be a const?

In the future should there be a strings / error msgs package?

Reply:
made it a const and put it at the top of cmdrunner.go file